src/components/Tag/core/tag.module.scss

```
@@ -0,0 +1,52 @@
+@use 'sass:map';
+@use '../../../style/variables/colors.scss';
+@use '../../../style/variables/spaces.scss';
+@use '../../../style/mixins/font-config.scss';
+
+.tag {
+  color: map.get(colors.$colors, 'base-white');
+  border-radius: map.get(spaces.$spaces, '1');
+
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  width: 52px;
+  height: 40px;
+  top: 20px;
+  left: 20px;
+  padding: 8px 12px;
```

The values are correct but you should use the variables : 
```suggestion
  padding: map.get(spaces.$spaces, '1') map.get(spaces.$spaces, '2');
```